Improve triggered output clock description (#2177)
modelica/fmi-standard: docs/2_2_common_mechanisms.adoc

[NOTE]
====

> <<outputClock,Output Clocks>> enable an FMU to provide a trigger to the outside world, e.g. to a <<triggered,triggered input Clock>> of another FMU. It is also allowd to assign variables to a triggered output clock.  +

Reviewer: ```suggestion
<<outputClock,Output Clocks>> enable an FMU to provide a trigger to the outside world, e.g. to a <<triggered,triggered input Clock>> of another FMU.
It is also allowed to assign variables to a triggered output clock.  +
```

> If an FMU only wants to actively trigger a model partition (Clock) of itself, it can also use a <<countdown-aperiodic-clock,countdown Clock>>.

Reviewer: this should be a normative note

Author: Why? It is only a recommendation ...

Reviewer 2: Sounds non-normative to me.

[NOTE]
====
<<outputClock,Output Clocks>> enable an FMU to provide a trigger to the outside world, e.g. to a <<triggered,triggered input Clock>> of another FMU.
It is also allowed to assign variables to a triggered output clock. +

> If an FMU only wants to actively trigger a model partition (Clock) of itself, it can also use a <<countdown-aperiodic-clock,countdown Clock>>.

Author: ```suggestion
If an FMU only wants to actively trigger a model partition (Clock) of itself, it might be more efficient to use a <<countdown-aperiodic-clock,countdown Clock>>.
```
